WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerView.xaml

```diff
@@ -65,14 +65,36 @@
                   </MenuItem>
                 </ContextMenu>
               </StackPanel.ContextMenu>
-              <DrawingPresenter Width="16" Height="16" Drawing="{DynamicResource WalletExplorer_ClosedWallet}" />
+              <StackPanel.DataTemplates>
+                <DataTemplate DataType="ViewModels:ClosedHardwareWalletViewModel">
+                  <DrawingPresenter Width="16" Height="16" Drawing="{DynamicResource WalletExplorer_HardwareWalletClosed}" />
+                </DataTemplate>
+                <DataTemplate DataType="ViewModels:ClosedWatchOnlyWalletViewModel">
+                  <DrawingPresenter Width="16" Height="16" Drawing="{DynamicResource WalletExplorer_WatchOnlyWalletClosed}" />
+                </DataTemplate>
+                <DataTemplate DataType="ViewModels:ClosedWalletViewModel">
+                  <DrawingPresenter Width="16" Height="16" Drawing="{DynamicResource WalletExplorer_WalletClosed}" />
+                </DataTemplate>
+              </StackPanel.DataTemplates>
+              <ContentControl Content="{Binding }" />
```

Comment:
This will Bind what?

Reply:
`{Binding PropertyName}` means bind to the thing that is contained by that property...
`{Binding }` or `{Binding .}` means bind to the whole object (`DataContext`)
http://avaloniaui.net/docs/binding/bindings